src/PlusDataCollection/vtkPlusGenericSerialDevice.cxx

```diff
@@ -170,6 +184,70 @@ PlusStatus vtkPlusGenericSerialDevice::InternalUpdate()
   return PLUS_SUCCESS;
 }
 
+PlusStatus vtkPlusGenericSerialDevice::SetDTR(bool OnOff)
+{
+  // Either update or send commands - but not simultaneously
+  PlusLockGuard<vtkPlusRecursiveCriticalSection> updateMutexGuardedLock(this->Mutex);
+
+  PlusStatus retval;
+
+  if (OnOff == DTR)
+  {
+    return PLUS_SUCCESS; //already the desired value
+  }
+  else if (OnOff == true)
+  {
+    retval = this->Serial->SetDTR();

```

Comment:
Instead of SetDTR() and ClearDTR() have a single method SetDTR(bool). It would make the code much shorter and simpler.